dojo/settings/settings.dist.py

```diff
@@ -1250,7 +1250,7 @@ def saml2_attrib_map_format(dict):
     'StackHawk HawkScan': ['vuln_id_from_tool', 'component_name', 'component_version'],
     'Hydra Scan': ['title', 'description'],
     'DrHeader JSON Importer': ['title', 'description'],
-    'PWN SAST': ['title', 'description'],
+    'PWN SAST': ['unique_id_from_tool'],
```

Comment:
I guess you can remove the line 'PWN SAST': ['unique_id_from_tool'], 

I made a PR regarding this unnecessary setting: https://github.com/DefectDojo/django-DefectDojo/pull/9188

Reply:
All set.  Thanks @manuel-sommer